insights/parsers/systemctl_show.py

```diff
@@ -164,7 +172,7 @@ class SystemctlShowTarget(SystemctlShowServiceAll):
         >>> 'network.target' in systemctl_show_target
         True
         >>> systemctl_show_target.get('network.target').get('WantedBy', None)
-        'NetworkManager.service'
+        ['NetworkManager.service']
```

Comment:
Same comment as above, I believe all values will be a `list` type.  Will this command actually return duplicates that require a list?